### ISSUE TITLE
Support single quotes

### ISSUE THREAD

**Issue by scythewyvern** [jonny-lake/tw-break#4]
There is currently no syntax highlighting in single quotes

<img width="383" alt="image" src="https://github.com/jonthemonke/tw-break/assets/98530570/407cdddd-ae24-4db0-b2b3-0bbf75de7757">

Is it possible to add support for it?
Will it work?
```diff
-  "injectionSelector": "L:string.quoted.double",
+  "injectionSelector": "L:string.quoted.double, L:string.quoted.single",
```

**Comment by jonthemonke** [comment]
This has been addressed in version 1.0.4. Please let me know if you have any other issues!

**Comment by scythewyvern** [comment]
> This has been addressed in version 1.0.4. Please let me know if you have any other issues!

its just amazing, thanks!!
